frontend/src/components/CookRecipe.js

```diff
@@ -92,12 +111,74 @@ class CookRecipe extends Component {
             </div>
           </Tab>
           <Tab eventKey="tutorial" title="Tutorial">
-            {/* to do */}
-            <div className="tab-content"></div>
+            <div className="tab-content">
+              <Carousel
+                interval={null} // to disable auto play of the carousel
+                onSelect={this.setSelectedStep}
+                defaultActiveIndex={this.getSelectedStep}
+                nextIcon={
+                  <img
+                    src={navigateNext}
+                    alt="next step"
+                    className="carousel-control"
+                  />
+                }
+                prevIcon={
+                  <img
+                    src={navigatePrevious}
+                    alt="previous step"
+                    className="carousel-control"
+                  />
+                }
+              >
+                {recipe.instructions.map((step, i) => (
+                  <Carousel.Item key={i} className="carousel-step">
+                    {step}
+                    <Button variant="" onClick={this.readStep}>
+                      <img src={speaker} alt="read the step" />
```

Comment:
I don't think we had this in the Figma prototype but IMO it'll make for a smoother experience if we reverse this behavior and making the button apply to the entire recipe.

So when the user starts the step by step tutorial, each step should be read out loud when the user navigates to it. There should be a crossed-out speaker button up top that turns this behavior off.

@lminchillo WDYT?

Reply:
Yes, it wasn't on the prototype and I don't think we put much thought into it, but it sounds like a good idea. @mam249 ?

Reply:
I implemented this